components/automate-cli/pkg/verifyserver/server/api/v1/s3config.go

```diff
@@ -0,0 +1,27 @@
+package v1
+
+import (
+	"fmt"
+
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/models"
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/response"
+	"github.com/gofiber/fiber"
+)
+
+func (h *Handler) GetS3Config(c *fiber.Ctx) {
+	s3ConfigRequest := new(models.S3ConfigRequest)
+	if err := c.BodyParser(&s3ConfigRequest); err != nil {
+		errString := fmt.Sprintf("s3 config request body parsing failed: %v", err.Error())
+		h.Logger.Error(fmt.Errorf(errString))
+		c.Status(fiber.StatusBadRequest).JSON(response.BuildFailedResponse(&fiber.Error{Message: err.Error()}))
+		return
+	}
+	s3Connection := h.S3ConfigService.GetS3Connection(*s3ConfigRequest)
```

Comment:
what if the s3 connection fails? when the connection fails, what is the need of checking the bucket access?

Reply:
s3 connection just gives session, it doesn’t validate your access key and secret key , to validate secret and access key we need to run list bucket

Reply:
Here you are returning the check response `S3ServiceCheck`, which has passed field, my question is, in case the passed value is false, do we still need to check the `GetBucketAccess`?